src/mds/Migrator.cc

```diff
@@ -1792,20 +1792,26 @@ void Migrator::encode_export_dir(bufferlist& exportbl,
     
     if (dn->get_linkage()->is_remote()) {
       // remote link
-      exportbl.append("L", 1);  // remote link
+      exportbl.append("l", 1);  // remote link
       
       inodeno_t ino = dn->get_linkage()->get_remote_ino();
       unsigned char d_type = dn->get_linkage()->get_remote_d_type();
+      ENCODE_START(2, 1, exportbl);
```

Comment:
Maybe we should add one parent class to do the decode/encode stuff, then let  both the `InodeStoreBase ` and `CDir` to inherit from it. Because the CDir is doing it by using its own method, it cannot use the CDentry's directly dues to the defer encoding feature will encode dentries in batch.